pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandlerTest.java

```diff
@@ -477,7 +669,13 @@ public void testRewriteRawForwardIndexForSingleColumn()
         SegmentDirectory.Writer writer = segmentLocalFSDirectory.createWriter();
 
         List<FieldConfig> fieldConfigs = new ArrayList<>(_tableConfig.getFieldConfigList());
-        FieldConfig config = fieldConfigs.remove(i);
+        int index = -1;
+        for (int j = 0; j < fieldConfigs.size(); j++) {
+          if (fieldConfigs.get(j).getName().equals(_noDictionaryColumns.get(i))) {
+            index = j;
```

Comment:
Thanks for fixing this.
Should we add a `break` here?

Reply:
done